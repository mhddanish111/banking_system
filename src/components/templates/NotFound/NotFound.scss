@import '../../../styles/spacing.scss';
.not-found {
    .main-container {
        display: flex;
        justify-content: center;
    }
    p {
        margin-top: $space-l;
    }
}