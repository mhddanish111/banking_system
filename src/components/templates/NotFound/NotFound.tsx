import React from 'react';
import Header from '../../organisms/Header';
import './NotFound.scss';

const NotFound: React.FC<any> = () => {
  return (
    <div className="not-found">
      <Header isDefault={false} />
      <main className="main-container">
        <p>404 Page Not Found !</p>
      </main>
    </div>
  );
};

export default NotFound;
