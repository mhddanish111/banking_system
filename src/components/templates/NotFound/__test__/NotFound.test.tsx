import { shallow } from 'enzyme';
import NotFound from '../NotFound';

describe('<NotFound />', () => {
  const wrapped = shallow(<NotFound />);
  it('should render the NotFound Component correctly', () => {
    expect(wrapped).toMatchSnapshot();
  });
});
