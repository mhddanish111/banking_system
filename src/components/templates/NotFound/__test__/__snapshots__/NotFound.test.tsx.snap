// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> should render the NotFound Component correctly 1`] = `
<div
  className="not-found"
>
  <Header
    isDefault={false}
  />
  <main
    className="main-container"
  >
    <p>
      404 Page Not Found !
    </p>
  </main>
</div>
`;
